walletunlocker/service.go

```diff
@@ -398,6 +406,13 @@ func (u *UnlockerService) InitWallet(ctx context.Context,
 			"non-negative", recoveryWindow)
 	}
 
+	// Ensure that the macaroon root key is *exactly* 32-bytes.
+	macaroonRootKey := in.MacaroonRootKey
+	if len(macaroonRootKey) != 0 && len(macaroonRootKey) != 32 {
```

Comment:
nit: somehow it's easier to read `len() > 0 && len() != 32`. But might just be me. We should replace the magic number by `macaroons.RootKeyLen` though.

Reply:
Looking around the codebase, I see roughly equal usage of `len() > 0 && len() != 32` and `len() != 0 && len() != 32` and I believe I copied this from one such case of the latter, but I'll change it here since it doesn't matter to me.